Library/Homebrew/cmd/audit.rb

```diff
@@ -423,7 +426,7 @@ def audit_homepage
   end
 
   def audit_github_repository
-    return unless @strict
+    return unless @online
```

Comment:
That feels a bit weird to me to have `--online` do nothing without `--strict` being passed. I can see us wanting to add some non-strict `--online` tests?